<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatTreeDlg</class>
 <widget class="QDialog" name="CatTreeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Categories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="btnAdd2Root">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>32</horstretch>
            <verstretch>32</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>32</horstretch>
            <verstretch>32</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../TM3res.qrc">
            <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDel">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../TM3res.qrc">
            <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add (Ins)</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete (Del)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
  </action>
  <action name="actionAdd2Root">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add to root</string>
   </property>
   <property name="toolTip">
    <string>Add to root (Ctrl+Ins)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../TM3res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAdd</sender>
   <signal>triggered()</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>addCat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDel</sender>
   <signal>triggered()</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>delCat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit</sender>
   <signal>triggered()</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>updateCat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd2Root</sender>
   <signal>triggered()</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>addRoot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>collapsed(QModelIndex)</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>collapse(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>expanded(QModelIndex)</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>expand(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CatTreeDlg</receiver>
   <slot>doubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>614</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCat()</slot>
  <slot>delCat()</slot>
  <slot>updateCat(QModelIndex)</slot>
  <slot>updateCat()</slot>
  <slot>addRoot()</slot>
  <slot>expand(QModelIndex)</slot>
  <slot>collapse(QModelIndex)</slot>
  <slot>doubleClicked(QModelIndex)</slot>
 </slots>
</ui>
